src/components/NewsArticleTeaser.tsx

```diff
@@ -19,19 +21,47 @@ export function NewsArticleTeaser({
   thumbnailURL,
 }: NewsArticleTeaserProps): JSX.Element {
   return (
-    <article>
+    <article sx={{ maxWidth: '500px', width: 'auto' }}>
```

Comment:
Jelen esetben érdemes inkább a `ch` mértékegységre hagyatkozni, mely a soronkénti karakterek számára utal. Ha ez mégsem volna szimpatikus _([IE 11](https://caniuse.com/#feat=ch-unit))_, akkor is jobb volna legalább `em` egységet használni, hiszen lehet, hogy valaki a böngészőjében megnöveli a 16px-es alapértelmezett betűméretet. `ch` vagy `em` használatával ilyenkor is fix marad a soronkénti betűk száma, míg `px`-et alkalmazva a nagyobb betűméretet használók számára túl kevés szöveg fog megjelenni egy-egy sorban.